diff --git a/src/CryptKey.php b/src/CryptKey.php
index bef2f3c..7816a32 100644
--- a/src/CryptKey.php
+++ b/src/CryptKey.php
@@ -70,7 +70,7 @@ class CryptKey
         if ($keyPermissionsCheck === true) {
             // Verify the permissions of the key
             $keyPathPerms = \decoct(\fileperms($this->keyPath) & 0777);
-            if (\in_array($keyPathPerms, ['400', '440', '600', '640', '660'], true) === false) {
+            if (\in_array($keyPathPerms, ['600', '660'], true) === false) {
                 \trigger_error(
                     \sprintf(
                         'Key file "%s" permissions are not correct, recommend changing to 600 or 660 instead of %s',
